public/index.pug

```diff
@@ -53,7 +53,16 @@ html(lang='en-US')
                   target='_blank'
                 ) #{ ext.nameAuthor }
     header.header
-      ul.header__list
+      .header__title-container
+        h1.header__title Simple Icons
+        p.header__description #{ iconCount } Free #[abbr(title='Scalable Vector Graphic') SVG] icons for popular brands
+      #menu-Toggle
+        svg(viewBox='0 0 100 80', width='40', height='40')
```

Comment:
We reallly need a wrapper container for the SVG? I think that the SVG can be the container itself. Also, the identifier breaks our style, it shouldn't be capitalized: `menu-toggle`.